extra-dev/packages/coq-sniper/coq-sniper.dev+8.11+8.12/opam

```diff
@@ -0,0 +1,36 @@
+opam-version: "2.0"
+maintainer: "Chantal.Keller@lri.fr"
+homepage: "https://github.com/smtcoq/sniper"
+dev-repo: "git+https://github.com/smtcoq/sniper.git"
+bug-reports: "https://github.com/smtcoq/sniper/issues"
+authors: ["Valentin Blot <Valentin.Blot@inria.fr>"
+          "Louise Dubois de Prsique <Louise.Dubois-de-Prisque@inria.fr>"
+          "Chantal Keller <Chantal.Keller@lri.fr"
+          "Pierre Vial <Pierre.Vial@inria.fr>"
+]
+license: "CECILL-C"
+build: [
+  [make "-j%{jobs}%"]
+]
+install: [
+  [make "install"]
+]
+depends: [
+  "ocaml" {>= "4.07.1" }
+  "coq" {>= "8.11~" & < "8.13~"}
+  "coq-metacoq"
+  "coq-smtcoq"
+]
+tags: [ 
+  "category:Computer Science/Decision Procedures and Certified Algorithms/Decision procedures"
+  "category:Miscellaneous/Coq Extensions"
+  "keyword: SMT"
+  "keyword: automation"
```

Comment:
It seems some .vo files get installed, so there should be a tag `"logpath:Sniper"`.